src/test/mir-opt/simplify_try/rustc.try_identity.SimplifyLocals.after.mir

```diff
@@ -3,22 +3,27 @@
 fn try_identity(_1: std::result::Result<u32, i32>) -> std::result::Result<u32, i32> {
     debug x => _1;                       // in scope 0 at $DIR/simplify_try.rs:5:17: 5:18
     let mut _0: std::result::Result<u32, i32>; // return place in scope 0 at $DIR/simplify_try.rs:5:41: 5:57
+    let _2: u32;                         // in scope 0 at $DIR/simplify_try.rs:6:9: 6:10
```

Comment:
huh why did these regress?

Reply:
The [SimplifyTry pass changes](https://github.com/rust-lang/rust/pull/73210#discussion_r457256085) didn't work right.